tools/systemd-units/char-rathena.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Rathena Char Server
+After=network-online.target remote-fs.target nss-lookup.target
+Wants=network-online.target
+#You might wanna edit the next line if your units are running in different servers instead of all in a single server.
+Requires=map-rathena.service login-rathena.service
```

Comment:
```suggestion
```
I think we should remove these dependencies. All of them should run independent of each other and only if installed.